tools/unitctl/unitctl/src/cmd/save.rs

```diff
@@ -0,0 +1,46 @@
+use crate::unitctl::UnitCtl;
+use crate::wait;
+use crate::UnitctlError;
+use crate::requests::send_empty_body_deserialize_response;
+use unit_client_rs::unit_client::UnitClient;
+use tar::{Builder, Header};
+use std::fs::File;
+
+
+pub async fn cmd(
+    cli: &UnitCtl,
+    filename: &String
+) -> Result<(), UnitctlError> {
+    if !filename.ends_with(".tar") {
+        eprintln!("Warning: writing tarball to {}", filename);
+    }
```

Comment:
Do we need to use a tarball?

If you just use `/` as the endpoint then you'll get everything, it's then very easy to use something like `jq(1)` to get just the bit you want.

It might also be useful to dump the config to stdout by default (if no filename is given) then you could do something like

```
$ unitctl save | jq '.certificiates'
```

Unless unitctl grew subcommands for just extracting certain bits of the config... it'd still be useful to be able to just dump to stdout for copy and pasting into bug reports etc...

Hmm, maybe `dump-cfg` or something would be a better name for this?

Reply:
The problem with dumping `/` is that you cannot import it again. So I see the benefit of separating `/config`, `/certificates` and `/js_modules` into separate artefacts.

Agree with stdout by default. So you could still do
```
unitctl save | tar xf - certificates.json
```

Reply:
I think the tarball is worthwhile here. Rarely will the User want to store a directory somewhere, and a directory is not something that can be distributed or shared without compression.

I also think it is an easy value add to not require the user to know how `jq` works.

Reply:
stdout as an output target is now implemented:
```
unitctl export -f - | tar xf - config.json
```

Reply:
Heh, think I'll stick to `jq </opt/unit/state/conf.json`

Reply:
You are welcome to. In fact you can use this code like this:
```
$ unitctl export -f - | tar xfO config.json | jq .....
```